src/neptune/new/handler.py

```diff
@@ -311,12 +316,13 @@ def log(
                 elif is_float_like(first_value):
                     attr = FloatSeries(self._container, parse_path(self._path))
                 elif is_string_like(first_value):
-                    warn_once(
-                        message="The object you're logging will be implicitly cast to a string."
-                        " We'll end support of this behavior in `neptune-client==1.0.0`."
-                        " To log the object as a string, use `.log(str(object))` instead.",
-                        stack_level=3,
-                    )
+                    if not from_stringify_value:
+                        warn_once(
+                            message="The object you're logging will be implicitly cast to a string."
+                            " We'll end support of this behavior in `neptune-client==1.0.0`."
+                            " To log the object as a string, use `.log(str(object))` or"
+                            " `.log(stringify_unsupported(collection))` for collections and dictionaries instead."
```

Comment:
Remove "instead" from here, it doesn't flow well. Add link to https://docs.neptune.ai/setup/neptune-client_1-0_release_changes